README.md

```diff
@@ -127,9 +127,7 @@ The basic common usage of Easegress is to quickly set up proxy for the backend s
 We can download the latest or history binaries from the [release page](https://github.com/megaease/easegress/releases). For example, we can install Easegress v1.0.0 for Linux amd64 platform with command:
```

Comment:
```suggestion
We can download the latest or history binaries from the [release page](https://github.com/megaease/easegress/releases). Following shell script will download and extract binaries to `./easegress` folder:
```

Reply:
sorry, I merge it too quickly... let me do this in another PR.